assignments/lambda-classes.js

```diff
@@ -1 +1,267 @@
 // CODE here for your Lambda Classes
+
+
+//CLASSES
+
+class Person {
+    constructor(persAttr) {
+        this.name = persAttr.name;
+        this.age = persAttr.age;
+        this.location = persAttr.location;
+        this.gender = persAttr.gender;
+    }
+    speak() {
+        console.log(`Hello, my name is ${this.name}, I am from ${this.location}`);
+    }
+};
+
+class Instructor extends Person {
+    constructor(instrAttr) {
+        super(instrAttr);
+        this.specialty = instrAttr.specialty;
+        this.favLanguage = instrAttr.favLanguage;
+        this.catchPhrase = instrAttr.catchPhrase;
+    }
+    demo(subject) {
+        console.log(`Today we are learning about ${subject}`);
+    }
+    grade(student, subject) {
+        console.log(`${student.name} recieves a perfect score on ${subject}`);
+    }
+    gradeStudent(student) {
+        student.grade += Math.round((Math.random()-.5)*20);
```

Comment:
This is a very efficient and clean way to write this method, best way I've seen it done yet!